R/kwic.R

```diff
@@ -159,9 +158,18 @@ print.kwic <- function(x, ...) {
 #' @export
 #' @method as.tokens kwic
 as.tokens.kwic <- function(x, ...) {
-    result <- attr(x, 'tokens')
-    names(result) <- x$docname
-    docvars(result) <- data.frame('_docid' = attr(x, 'docid'),
-                                  '_segid' = attr(x, 'segid'))
+    vars <- attr(x, 'docvars')
```

Comment:
This not only makes no real sense to me, but also violates the idea that docnames should be unique. What makes more sense is to expand the docnames as you do with the results of 

```r
head(rownames(attr(as.tokens(testkwic), "docvars")))
## [1] "1789-Washington.1" "1789-Washington.2" "1797-Adams.1"      "1797-Adams.2"      "1797-Adams.3"      "1805-Jefferson.1" 
```

But is it meaningful to coerce kwic objects to tokens?  What is the use case?